networktest.go

```diff
@@ -245,34 +245,38 @@ type networkHarness struct {
 	watchRequests chan *watchRequest
 }
 
-// newNetworkHarness creates a new network test harness given an already
-// running instance of btcd's rpctest harness. Any extra command line flags
-// which should be passed to create lnd instance should be formatted properly
-// in the lndArgs slice (--arg=value).
-// TODO(roasbeef): add option to use golang's build library to a binary of the
-// current repo. This'll save developers from having to manually `go install`
-// within the repo each time before changes.
-func newNetworkHarness(lndArgs []string) (*networkHarness, error) {
+/*
```

Comment:
The godoc-like comment for this function shouldn't be removed. I like that the second sentence along with the `TODO` to myself was shifted to the method below, but can you please restore the original first sentence. Thanks!